charts/metrics-server/values.yaml

```diff
@@ -146,3 +146,5 @@ topologySpreadConstraints: []
 
 # Annotations to add to the deployment
 deploymentAnnotations: {}
+
+deploymentSchedulerName: ""
```

Comment:
I think we could just go with `schedulerName` as there shouldn't be ambiguity around the context here.

Reply:
modified,thanks